content/en/docs/release-oversight/test-case-analyzer.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Test Case Analyzer"
+description: Introduction of a new tool to analyze test cases across multiple job runs.
+---
+
+### Summary
+On top of the existing aggregator framework, the new Test Case Analyzer tool is introduced to analyze specific test results from a much broader set of candidate job runs, without requiring the entire job to pass.
+
+### Improvement to CI Payload Signals
+CI and nightly payloads run many different jobs. 
+Some of those jobs are blocking while the majority are informing. 
+Failure of any blocking jobs will reject the payload, but informing job results are not utilized in any automated way to provide valuable CI signals. 
+Since the informing job results still carry lots of information about the health of the system, it would be valuable to utilize those results in an automatic way to provide further improvement to the CI Signals.
+
+The Test Case Analyzer was created to expand our ability to block payloads by analyzing test cases from job runs belonging to multiple different jobs, regardless of whether they are informing or blocking. 
+The job runs can be selected by certain variants of interest (e.g platform, network, or infrastructure). 
+Out of the selected group of job runs, one or a subset of the tests can be analyzed using certain criteria, the job runs themselves do not have to fully pass. 
+One example criteria is to have a minimum required number of test successes. 
+The end result of the analyzer can be used to form a blocking signal for the overall payload.
+
+This gives us the ability to gate a payload run with information solely from informing jobs. 
+
+### Use Cases
```

Comment:
I think this drives a section that describes the parameters you added to the command to explain how to identify runs for a given payload, how to identify the variants you want to select on, how to decide what tests to select, and how to describe selection criteria.  Think of it in terms of how you'd use use-cases to defend a proposal in a template like: https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md#proposal